### PR TITLE
vt_utils/sys_time.py: Add a new utility for the time

### PR DESCRIPTION
**Pull request by nickzhq**
Here's a new time module including the functions about time.

### CODE REVIEW

**Comment by nickzhq** [comment]
@YongxueHong @zhencliu @yanan-fu @luckyh  Please take a look, thanks!

File: virttest/vt_utils/sys_time.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+             this function is NOT available.
+    """
+    # for Linux system
+    if os.name == "posix":
+        timezone_cmd = 'timedatectl | grep "Time zone"'
+        timezone_re = '^(?:\s+Time zone:\s)(\w+\/\S+|UTC)(?:\s\(\S+,\s)([+|-]\d{4})\)$'
+    else:
+        raise OSError("The function was NOT available on this system yet!")
+
+    timezone = process.run(timezone_cmd, shell=True).stdout_text
```
Comment (zhencliu) [inline virttest/vt_utils/sys_time.py:40]: Note that we might hit a CmdError exception here(ignore_status=False by default if I remember it correctly)

**Comment by nickzhq** [comment]
@YongxueHong @zhencliu @yanan-fu @luckyh Please take a look, thanks!

**Comment by zhencliu** [comment]
LGTM

BTW: what about systime instead of sys_time? @nickzhq @YongxueHong

**Comment by YongxueHong** [comment]
> LGTM
> 
> BTW: what about systime instead of sys_time? @nickzhq @YongxueHong

Hi @zhencliu 
I prefer the `sys_time`(system time)

**Review by YongxueHong — APPROVED** [review]
LGTM.

**Comment by nickzhq** [comment]
@yanan-fu @luckyh Please take a look, thanks!

**Review by yanan-fu — APPROVED** [review]
Ack

**Pull request merged**